src/components/fetcher/index.js

```diff
@@ -62,14 +68,14 @@ const Fetcher = ({
             }
           } catch (error) {
             fetchErrorCounter++;
-            console.error('fetchMam error 1', fetchErrorCounter, data.length, error);
+            console.error('fetchMam error 1', fetchErrorCounter, error );
```

Comment:
Solutions like `fetchErrorCounter` are most likely no longer needed. These were in place as a workaround for an unstable network in the past